OWNERS_ALIASES

```diff
@@ -133,14 +133,12 @@ aliases:
     - childsb
 
   sig-scheduling-maintainers:
-    - bsalamat
```

Comment:
@mrbobbytables As we have `emeritus_approvers`, but I didn't see a pattern how it's used in root OWNERS files. Should we specify "<sig-xyz-emeritus_approvers>"? (as a top lever "emeritus_approvers" doesn't quite make sense")

Reply:
Sorry for the long delay in response, I didn't catch the mention till now. >_<
We don't currently have a good convention for emeritus in `OWNERS_ALIASES`. In the past folks have just been removed. 

I'm open to adding some guidelines for adding it `OWNERS_ALIASES`, but that should wind up being a separate discussion.

For now to avoid blocking on the topic, I'd comment bsalamat out for now like this:
```
sig-scheduling-maintainers:
#  -  bsalamat - emeritus
```
We have a few other instances with folks in there commented out specifically for manually escalation only.